docs/Database.md

```diff
@@ -5,6 +5,46 @@ The `CCT` and `MyGordon` databases exist in:
 - `admintrainsql.gordon.edu` - The development/testing database server
 - `adminprodsql.gordon.edu` - The live production database server.
 
+### Updating context tables (_new columns, tables, procedures..._)
+
+SQL Table models are handled in `360api` by Entity Framework Power Tools (`efpt`). To script in new tables do the following:

```

Comment:
This is really an oversimplification. Entity Framework Core is an ORM that handles mapping SQL tables/views/stored procedures to C# Objects, and transforming C# code into SQL Commands.

EF Core Power Tools is just a Visual Studio extension that adds a UI and some helpful reverse-engineering niceties on top of EF Core.